### PR TITLE
navbar

### PR DESCRIPTION
**Pull request by Yagna24**
Added the files.

### CODE REVIEW

**Review by arps18 — CHANGES_REQUESTED** [review]
The HTML and CSS file are not working correctly, kindly look into the issue!

File: navbar/index.html [hunk 1]
```diff
@@ -0,0 +1,262 @@
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link rel = "stylesheet"  href = "/home/yagna/Desktop/proj/healthier-ai.github.io-master/navbar/index.css" type="text/css">
```
Comment (arps18) [inline navbar/index.html]: Set proper Path!

File: navbar/index.html [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+<style>
```
Comment (arps18) [inline navbar/index.html]: Separate CSS File

**Review by arps18 — CHANGES_REQUESTED** [review]

File: navbar/index.html (hunk 2 above)
Comment (arps18) [reply navbar/index.html]: Screenshot (33)

Kindly resolve the same!

**Review by arps18 — CHANGES_REQUESTED** [review]

**Comment by AyushBherwani1998** [comment]
Hey @Yagna24, thanks for contribution, any updates on the requested changes?

**Comment by Yagna24** [comment]
> Hey @Yagna24, thanks for contribution, any updates on the requested changes?

Updated the files , setting up mobile view for the same atm.

**Comment by AyushBherwani1998** [comment]
@Yagna24 thanks for the quick updates on the PR. Can you please attach a screenshot for mobile view and desktop view?

**Comment by Yagna24** [comment]
> 
> 
> @Yagna24 thanks for the quick updates on the PR. Can you please attach a screenshot for mobile view and desktop view?

Yeah , sure 
Screenshot (7)
Screenshot (8)

**Comment by arps18** [comment]
> > @Yagna24 thanks for the quick updates on the PR. Can you please attach a screenshot for mobile view and desktop view?
> 
> Yeah , sure
> Screenshot (7)
> Screenshot (8)

- Kindly Work on the font section as well and try the same in the current index.html file on the repo!

**Pull request closed without merging**